### ISSUE TITLE
bug: "Override path to python" not working

### ISSUE THREAD

**Issue by steakhouserodriguez** [Opentrons/opentrons#15138]
### Overview

The setting has no effect, even after restarting the app and computer.

### Steps to reproduce

<img width="656" alt="image" src="https://github.com/Opentrons/opentrons/assets/205124/4cf768b0-4d39-452b-90fe-2d9291304630">

```
#otpython.py
metadata = {
    "protocolName": "otpython",
    "apiLevel": "2.17",
}

import sys
raise Exception(sys.executable)

def run(ctx):
    pass

```

### Current behavior

protocol analysis should use my interpreter, not the bundled.

![image](https://github.com/Opentrons/opentrons/assets/205124/8ea6fb5b-f6f1-446d-8829-b9d07660a941)

### Expected behavior

Should be 
```
/opt/miniconda3/envs/ivynatal_lab/bin/python3
```

### Operating system

Mac

### System and robot setup or anything else?

I'm using miniconda? 
App version 7.2.2

**Comment by koji** [comment]
@steakhouserodriguez 
Could you check one thing?

There is the config file of the app and the following is the file path.

Library/Application\ Support/Opentrons/config.json

**Comment by steakhouserodriguez** [comment]
yeah, config.json has the path I submitted. I've just emailed support@.

**Comment by koji** [comment]
@steakhouserodriguez 

if you type `which python`, is the output path is the same as the path you submitted?

**Comment by steakhouserodriguez** [comment]
Hmm, I've ended up setting the environment to `envs/ivynatal_lab` instead of `envs/ivynatal_lab/bin` and it seems to work now. I'm 75% sure I did try this in the past without success, but I can't reproduce the issue anymore.


> @steakhouserodriguez
> 
> if you type `which python`, is the output path is the same as the path you submitted?

I'm not sure if its relevant at this point, but the output paths would correspond to the environment's python path. I don't have python installed in system path.
